### DISCUSSION TITLE
Add geometry tools

### DISCUSSION THREAD

**Discussion by VasigaranAndAngel** [geovens/gInk#114] [Ideas]
This is a cool tool. Adding some geometry tools will make this tool better. Like, Line drawer (Show the length of the line when drawing), Ruler (Showing length and snapping pen or highlighter drawings), Compass (for draw circle. show the length of Radius), Protractor. Transparent grid lines. Snapping with all the points. and I have more ideas. I like to make everything the most advanced.

**Comment by pubpub-zz** [comment]
Hi @VasigaranAndAngel , 
Tools may not be exactly as you would like but you could be interested:
https://github.com/pubpub-zz/ppInk/releases/tag/1.6RC1
